providers/hns/hns_roce_u_hw_v2.c

```diff
@@ -1040,7 +1039,9 @@ static int hns_roce_u_v2_post_recv(struct ibv_qp *ibvqp, struct ibv_recv_wr *wr,
 static void __hns_roce_v2_cq_clean(struct hns_roce_cq *cq, uint32_t qpn,
 				   struct hns_roce_srq *srq)
 {
+	int wqe_index;
```

Comment:
is it really signed?

Reply:
Thank you.
wqe_index should be uint16_t, I have modified it and re-pushed.